api/comments/views.py

```diff
@@ -79,6 +79,7 @@ class CommentDetail(JSONAPIBaseView, generics.RetrieveUpdateDestroyAPIView, Comm
         is_abuse       boolean            has this comment been reported by the current user?
         has_children   boolean            does this comment have replies?
         can_edit       boolean            can the current user edit this comment?
+        new_mentions   list               guids of mentioned users in content of comment
```

Comment:
Why is it `new_mentions` in the API instead of just `mentions`? It looks like we actually have `old_mentions` as well:

![screen shot 2016-04-29 at 12 40 05 pm](https://cloud.githubusercontent.com/assets/134455/14922810/9ff6de42-0e07-11e6-9fc1-6d3dba55dc8a.png)

Why aren't we documenting that?